lib/radbeacon/le_scanner.rb

```diff
@@ -7,9 +7,26 @@ def initialize(duration = 5)
       @duration = duration
     end
 
+    def scan_command
+      rout, wout = IO.pipe
+      scan_command_str = "sudo hcitool lescan"
+      pid = Process.spawn(scan_command_str, :out => wout)
+      begin
+        Timeout.timeout(@duration) do
+          Process.wait(pid)
+        end
+      rescue Timeout::Error
+        Process.kill('INT', pid)
```

Comment:
Ok so my thinking was that I wanted to use the gentlest signal to end the scan process with the hopes that it would keep the dongle from locking up. Does that make sense at all? I'm currently seeing in testing that with this method the `lescan` process isn't always terminating so perhaps I'm being too gentle. I'll switch it to `TERM`